src/main/js/PullRequestDetails.tsx

```diff
@@ -327,6 +333,7 @@ class PullRequestDetails extends React.Component<Props, State> {
             loading={mergeButtonLoading}
             repository={repository}
             pullRequest={pullRequest}
+            modalClose={() => this.mergeModalClosed()}
```

Comment:
```suggestion
            modalClose={this.mergeModalClosed}
```

Reply:
Ah yeah I forgot the babbel plugin that fixes `this`-bindings.